### PR TITLE
Fix/android/example testing

### LINKED ISSUE #10: Configure Microblink Linking client in Android
As a developer
I need to create a wrapper for Microblink Linking client
So that I can initialize the Linking client

### What it is now
The developer needs to create the client calling Microblink SDK directly

### What it should be
The developer calls TIKI receipt capacitor to initialize Microblink Linking client

### Accpetance criteria
TIKI receipt capacitor inititlizes the Microblink Linking Client in Android

### LINKED ISSUE #289: [Android] debug imap login error
### Reason

IMAP email is not working

### Description

Microblink is return an error when trying login. We need test all scenarios to had a view of whats is working and now in our code, and microblink code.

## TestFlight App 
- [x] Test Login Email Account US
- [x] Test Login-Scan Email Account US
- [x] Test Scan Email Account US
- [x] Test Login Email Account BR (work in account BR only if the language of account is in english)
- [x] Test Login-Scan Email Account BR
- [x] Test Scan Email Account BR

## POC
- [x] Test Login Email Account US
- [x] Test Login-Scan Email Account US
- [x] Test Scan Email Account US
- [x] Test Login Email Account BR
- [x] Test Login-Scan Email Account BR
- [x] Test Scan Email Account BR

## Capacitor 
- [x] Test Login Email Account US
- [x] Test Login-Scan Email Account US
- [x] Test Scan Email Account US
- [x] Test Login Email Account BR
- [x] Test Login-Scan Email Account BR
- [x] Test Scan Email Account BR
### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Schuler-Gabriel**
# Issues Addressed

Fixes #289

### CODE REVIEW

File: android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Email.kt [hunk 1]
```diff
@@ -70,7 +69,7 @@ class Email {
         }
         val isImapInitialized = CompletableDeferred<Unit>()
         imapClient =
-            ImapClient(context, OnInitialize(isImapInitialized, onError)).apply { dayCutoff(30) }
+            ImapClient(context, OnInitialize(isImapInitialized, onError)).apply { dayCutoff(5) }
```
Comment (ricardobrg) [inline android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Email.kt:72]: daycutoff should be a parameter of scan

File: src/receipt-capture-plugin.ts [hunk 2]
```diff
@@ -9,15 +9,15 @@ import type { Receipt } from './receipt';
...
+  username?: string;
+  password?: string;
+  source?: string;
 }
 
 export interface ReqInitialize {
   licenseKey: string;
   productKey: string;
-  googleClientId: string | undefined;
+  googleId: string | null;
```
Comment (ricardobrg) [inline src/receipt-capture-plugin.ts:20]: remove googleId

File: example/src/main.ts [hunk 3]
```diff
@@ -1,40 +1,69 @@
...
 
 const plugin: ReceiptCapturePlugin = registerPlugin<ReceiptCapturePlugin>('ReceiptCapture', {
   web: () => import('../../src/receipt-capture-web').then((m) => new m.ReceiptCaptureWeb()),
 });
 
-const instance = new ReceiptCapture(plugin);
+const instance: ReceiptCapture = new ReceiptCapture(plugin);
+
+export const login = async (username: string, password: string, source: string) => {
+  console.log('login', username, password, source)
```
[the remark this answers is not in the document]
Comment (timoguin) [reply example/src/main.ts:16]: Let's remove password from the log.

[the remark this answers is not in the document]
Comment (ricardobrg) [reply example/src/main.ts:16]: done in #308 https://github.com/tiki/capture-receipt-capacitor/commit/c360bbbe69bdefc6b4ddeaef55e8a4940ee9491b

**Pull request closed without merging**